low-level/sensors/impl/generic/node_hw.py

```diff
@@ -48,6 +49,7 @@
 from message_handlers.node_data_msg_handler import NodeDataMsgHandler
 from sensors.INode_hw import INodeHWsensor
 from framework.utils.ipmi_client import IpmiFactory
+from sensors.impl.generic.node_hw_alerts_info import alert_for_event, Alert
```

Comment:
Seems 'Alert' is unused import, can we remove it.

Reply:
yes done